psk/server-psk-nonblocking.c

```diff
@@ -84,26 +85,37 @@ int main()
     char   buff[MAXLINE];  /* buffer for tcp connection */
     char   buf[MAXLINE];   /* string read from client */
     char   response[] = "I hear ya for shizzle";
+    char suites[]   =
```

Comment:
not sure that an array like this will work? when building and running I get a fatal error unable to set cipher list.

```
$ ./server-psk-nonblocking 
Fatal error : server can't set cipher list
```

Reply:
Ok this was a user error on my side. It works.